recipe/meta.yaml

```diff
@@ -11,24 +11,35 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  script: python -m pip install --no-deps --ignore-installed .
+  number: 2
+  noarch: python
+  # disable if already installed
+  script: (jupyter-nbextension disable qgrid --py --sys-prefix || true) && python -m pip install --no-deps --ignore-installed . && jupyter-nbextension enable qgrid --py --sys-prefix
```

Comment:
This does not appear to work for enabling qgrid automatically...I tried building this recipe locally and qgrid was not enabled after installing from the generated conda package (and it was enabled when I tried with a package that was built from the recipe on master).  

Looking at this again it looks like this `script:` option is in the `build:` section which means this will run as part of the build script on our CI services (travis, etc).  I don't think that's what we want because we need the `jupyter-nbextension enable` command to run on each qgrid user's machine after they install qgrid.

Unless you have something else you want to try, maybe we can just revert the `noarch` portion of this and keep the automated testing portion?